torch/_dynamo/variables/misc.py

```diff
@@ -1716,7 +1716,10 @@ def call_function(self, tx: "InstructionTranslator", args, kwargs):
         if tx.export:
             # For export cases, we can just make debugging functions no-ops
             return
-
+        # If this function is explicitly configured to be ignored, treat it
+        # as a no-op and do not run it or record it for reordering.
+        if self.value in torch._dynamo.config.ignore_logging_functions:
+            return
```

Comment:
We should be returning something here - probably `ConstantVariable.create(None)` (and elsewhere in this function)